src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs

```diff
@@ -239,7 +239,15 @@ public string GetTargetPath(string path, string name, string type, List<SQLDataD
             List<string> pathParts = new List<string>(path.Split(new string[] { "\\" }, StringSplitOptions.None));
             int len = pathParts.Count();
             List<string> fileNameParts = new List<string>(pathParts[len - 1].Split(new string[] { "." }, StringSplitOptions.None));
-            string newFileName = name + "_" + offset.ToString() + "." + fileNameParts[1];
+            string newFileName = "";
+            if (fileNameParts.Count == 2)
```

Comment:
Can you write a unit test for this?

Reply:
@isra-fel The present test i.e. `TestAzureVmWorkloadFullRestore` covers this scenario.

Reply:
Does it cover the new logic you added? Where `filenameparts.count != 2`

Reply:
Added